recipes/pyflame/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "pyflame" %}
+{% set version = "1.5.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
```

Comment:
Please use a tarball instead of the git repo:

https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos